.circleci/config.yml

```diff
@@ -120,12 +120,13 @@ jobs:
   deploy:
     working_directory: ~/openreview-py-repo
     docker:
-      - image: circleci/python:2.7
+      - image: cimg/python:3.7.14-node
     steps:
       - checkout
-      - run: pip install twine
-      - run: python setup.py bdist_wheel --universal
-      - run: twine upload dist/* --skip-existing
+      - run:
+          name: Deploy to dev instance
+          command: |
+            curl -X POST -H "Accept: application/vnd.github.v3+json" -H "Authorization: Bearer $OPENREVIEW_GITHUB" https://api.github.com/repos/openreview/openreview-py/dispatches -d '{"event_type":"openreview-py-updated"}'
```

Comment:
The command here is a "dispatch event" called "openreview-py-updated". The "openreview-py-updated" is saved in the action that allows to deploy to the dev site manually. It's the same process that we use to run openreview-web and openreview-py tests when an openreview-api-v1 branch is merged to master.